osu.Game/Utils/ZipUtils.cs

```diff
@@ -9,6 +9,34 @@ namespace osu.Game.Utils
 {
     public static class ZipUtils
     {
+        public static bool IsZipArchive(Stream stream)
```

Comment:
Thoughts on enforcing a random access stream type here?

```suggestion
        public static bool IsZipArchive(MemoryStream stream)
```

Not a huge concern but feels safer (not all streams are created equal, as some cannot be seeked).

Then again if this is kept as `Stream` then you could use this method in the `string` overload to reduce duplication... Not sure.

Reply:
Wasn't sure about that one. Probably safest to limit it until a usage is required otherwise?